server/imports/github-methods.js

```diff
@@ -133,3 +134,42 @@ export function putPageContent (userId, fullName, pageName, cb) {
   }, cb)
 
 }
+
+export function tagGhPages (userId, fullName, cb) {
+  let github = githubInterface(userId)
+
+  async.waterfall([
+    // Get current project version
+    Meteor.bindEnvironment(function (cb) {
+      github.getFileContents(fullName, 'package.json', cb)
+    }),
+    // Increment version
+    Meteor.bindEnvironment(function (res, cb) {
+      let pkg = JSON.parse(base64.decode(res.content))
+      pkg.version = incrementPatch(pkg.version)
+      cb(null, pkg, res.sha)
+    }),
```

Comment:
Yes totally agree, this code is very much minimum viable! Also, we want to disable the publish button if what's been tagged hasn't changed.